tests/Fixer/ArrayNotation/ArraySyntaxFixerTest.php

```diff
@@ -25,9 +25,9 @@
 {
     public function testInvalidConfiguration()
     {
-        $this->setExpectedException(
+        $this->setExpectedExceptionRegExp(
             'PhpCsFixer\ConfigurationException\InvalidFixerConfigurationException',
-            '[array_syntax] Invalid configuration: The option "a" does not exist.'
+            '#^\[array_syntax\] Invalid configuration: The option \"a\" does not exist\. Defined options are: \"syntax\"\.$#'
```

Comment:
Side question: why do you escape `"` characters? As far as I know, they are not meta-characters in PCRE patterns.